_data/education.yml

```diff
@@ -3,4 +3,4 @@
   dates: 2009
   qualification: B.S. in Computer Science
   description: | # this will include new lines to allow paragraphs
-    Focused on various Computer Science topics, my favorites include `Virtual Worlds`, `Topics in Game Production`, `Computer Graphics` and `Compilers`.
+    Focused on various Computer Science topics. My favorites include `Virtual Worlds`, `Topics in Game Production`, `Computer Graphics` and `Compilers`.
```

Comment:
`upstream` implies that favorites focused.

Reply:
👍  - although maybe there should be a comma after `Computer Graphics`? 🤷‍♂️